eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -76,6 +87,30 @@ jobs:
           artifactName: packages
           path: $(Build.ArtifactStagingDirectory)
 
+      - script: |
+          npm i -g typedoc
+        displayName: "Install typedoc"
+
+      - script: |
+          npm install
+        workingDirectory: $(System.DefaultWorkingDirectory)/eng/tools/generate-doc
+        displayName: "Install tool dependencies"
+
+      - pwsh: |
+          Write-Host "${{parameters.DocArgs}}"
+          node $(Build.SourcesDirectory)/eng/tools/generate-doc/index.js --dgOp "dg" -i "inc" --inc "${{parameters.DocArgs}}"
+        displayName: "Run Typedoc Docs"
+
+      - pwsh: $(Build.SourcesDirectory)/eng/tools/compress-subfolders.ps1 "$(Build.SourcesDirectory)/docGen" "$(Build.ArtifactStagingDirectory)/Documentation"
```

Comment:
for consistency can you put `|` here and move the commands to the next line.

Reply:
done